graph2tac/tfgnn/models.py

```diff
@@ -803,29 +804,53 @@ def __init__(self,
         super().__init__(name=name, **kwargs)
         self._hidden_size = hidden_size
 
+        if name_layer is None:
+            self._name_layer = None
+        else:
+            self._name_layer_core = tf.keras.layers.deserialize(name_layer)
+            self._name_layer = tf.keras.Sequential([
+                tf.keras.layers.Input(shape=[None], dtype=tf.float32, ragged=True),
+                tf.keras.layers.Embedding(
+                    input_dim=Dataset.MAX_LABEL_TOKENS,
```

Comment:
I'm worried about you getting this number from here.  The size we should use should be based on whatever that constant was from when we trained the model.  I would put this number in the dataset graph constants and get it from there.  (Or if we put the tokenizer in the model, you could get it from the tokenizer.)